pkgs/by-name/ze/zed-editor/package.nix

```diff
@@ -34,6 +34,8 @@
   git,
   apple-sdk_15,
   darwinMinVersionHook,
+  makeWrapper,
+  nodejs_23,
```

Comment:
Does it have to be a non LTS version of Node? The uneven releases aren't supported for a long time. Ideally we'd just use nodejs that gets updated automatically.

Reply:
No, the Zed code only checks that the major version is >18. I'm not exactly familiar with node packaging so I just grabbed the first result of the package search. I'll change it to `nodePackages_latest.nodejs`.